packages/twenty-server/src/database/commands/upgrade-version-command/upgrade.command.ts

```diff
@@ -237,7 +239,7 @@ export class UpgradeCommand extends UpgradeCommandRunner {
 
     const commands_140: VersionCommands = {
       beforeSyncMetadata: [],
-      afterSyncMetadata: [],
+      afterSyncMetadata: [this.populateMessageFolderFieldsCommand],
```

Comment:
revert :) let's not add it yet, we will add this to 1.5 actually (as 1.4 is out, we can't modify it once 1.4.0 is published, otherwise some selfhosters will miss it)